Skype/WebSDK/docs/PTVideoGroupGoogleChrome.md

```diff
@@ -0,0 +1,38 @@
+
+# Audio/Video in Google Chrome
+
+
+
+ _**Applies to:** Skype for Business 2015_
+
+At the moment we only support one incoming video channel in Google Chrome. 
+To still render video of multiple participants in a conversation we have introduced the **Active Speaker** API.
+
+Through the API the video of the currently speaking participant is sent over the one available channel.
+
+## Active Speaker API
+
+### How fo I know when to use the Active Speaker API?
```

Comment:
Maybe include a little text here explaining, "If you want your app to support group video on either Chrome or Microsoft Edge or IE, you will need to add code to detect what APIs to use when rendering a group av call."